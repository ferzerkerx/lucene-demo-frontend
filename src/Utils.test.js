import { checkStatus } from './Utils';

it('should throw error when response is not successful', () => {
  function check() {
    checkStatus({ status: 400 });
  }
  expect(check).toThrowErrorMatchingSnapshot();
});

it('should return successful response', () => {
  let response = { status: 200 };
  expect(checkStatus(response)).toBe(response);
});
