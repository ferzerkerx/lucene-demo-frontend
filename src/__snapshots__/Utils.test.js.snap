// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should throw error when response is not successful 1`] = `""`;
